delaymodel.py

```diff
@@ -274,19 +305,19 @@ def calculate_delay(self, publish : bool = True):
                 self.delay_data["delay_ns"] = (delay2*1e9).tolist()
                 self.delay_data["delay_rate_nsps"] = (rate*1e9).tolist()
                 self.delay_data["delay_raterate_nsps2"] = (raterate*1e9).tolist()
-                self.delay_data["effective_lo_0_mhz"] = self.lo_eff[0]
-                self.delay_data["effective_lo_1_mhz"] = self.lo_eff[1]
-                self.delay_data["sideband_0"] = self.sideband[0]
-                self.delay_data["sideband_1"] = self.sideband[1]
+                self.delay_data["effective_lo_0_mhz"] = sslo[0]
```

Comment:
Do `sslo` and `sideband` have default values in the event we get here via the first elif (the first ever command received is one with an invalid load time). If this happens I think the method will crash with undefined variable